### DISCUSSION TITLE
No installation possible at Ubuntu 22.04.2 LTS

### DISCUSSION THREAD

**Discussion by kaengo** [Akaflieg-Freiburg/enroute#324] [Q&A] [answered]
sudo snap install enroute-flight-navigation
error: cannot perform the following tasks:
- Ensure prerequisites for "enroute-flight-navigation" are available (cannot install prerequisite "kde-frameworks-5-91-qt-5-15-3-core20": no snap revision available as specified)

These prerequisites can not be installed by snap (or apt-get)

Greetings,
Holger

**Comment by kebekus** [comment, marked as answer]
@kaengo Now, this is a somewhat tragic point. I have been struggling with the technical shortcomings of the snap system for a very long time. For the last six months, I have been unable to compile the app once, even though I invested a lot of time. I have requested to remove that app from the Snap store, but Ubuntu has refused my request … probably out of fear that too many developers will abandon their platform.

I recommend using the flatpak binary provided on flathub.org, which runs smoothly and receives frequent updates.

- Setting up flatpak on Ubuntu: https://flatpak.org/setup/Ubuntu
- Link to the app in the flathub store: https://flathub.org/de/apps/de.akaflieg_freiburg.enroute

When I find the time, I will replace the Snap with a dummy package that opens a window with a message pointing the user to the flatpak.

I am sorry that I have no better news. If you know how to navigate the Snap ecosystem and would like to help, please let me know.

**Reply by kaengo** [reply]
Thank you, I did, as you recommanded!
